eslint.config.js

```diff
@@ -54,7 +55,18 @@ export default [
 			'no-console': 'off',
 
 			// Todo: do we want these?
-			'@typescript-eslint/no-unused-vars': 'off',
+			'@typescript-eslint/no-unused-vars': [
+				'error',
+				{
+					args: 'all',
+					argsIgnorePattern: '^_',
+					caughtErrors: 'all',
+					caughtErrorsIgnorePattern: '^_',
+					destructuredArrayIgnorePattern: '^_',
+					varsIgnorePattern: '^_',
+					ignoreRestSiblings: true,
+				},
+			],
```

Comment:
What's the reason for enabling the rule? The idea is to complement Biome with `typescript-eslint`, but it doesn't make much sense to enable two rules that do the same thing

Reply:
I tried with biome only first but it didn't catch many cases that eslint catched so I decided to enable both. If you have a better suggestion, happy to change this!

Reply:
If so, then we should switch off the Biome rule

Reply:
I think we should leave it enabled for Biome because they seem to match different things actually (eg. biome matches unused type parameters)